controllers/dataprotection/utils.go

```diff
@@ -199,7 +199,7 @@ func GetTargetPods(reqCtx intctrlutil.RequestCtx,
 	if err != nil {
 		return nil, err
 	}
-	if selector.Strategy == dpv1alpha1.PodSelectionStrategyAll || targetPods != nil ||
+	if selector.Strategy == dpv1alpha1.PodSelectionStrategyAll || len(targetPods) > 0 ||
```

Comment:
should also add explanation to `fallbackLabel` and `fallbackSelector` that it only take effect when  selector.Strategy equals to dpv1alpha1.PodSelectionStrategyAny

Reply:
already did it. 
```
// Specifies the fallback role to select one replica for backup, this only takes effect when the
	// `strategy` field below is set to `Any`.
	//
	// +optional
	FallbackRole string `json:"fallbackRole,omitempty"`
```